jiant/models.py

```diff
@@ -623,34 +635,36 @@ def build_pair_attn(d_in, d_hid_attn):
 
     # Build the "pooler", which does pools a variable length sequence
     #   possibly with a projection layer beforehand
-    if params["attn"] and not model.use_bert:
+    if params["attn"] and not model.use_pytorch_transformers:
         pooler = Pooler(project=False, d_inp=params["d_hid_attn"], d_proj=params["d_hid_attn"])
         d_out = params["d_hid_attn"] * 2
     else:
         pooler = Pooler(
-            project=not model.use_bert,
+            project=not model.use_pytorch_transformers,
```

Comment:
Why do we not this here? Don't we want to project before pooling in the else case?

Reply:
The current logic is to optionally project with non-transformers (depends on other params elsewhere), and never project with transformers. I don't love it, because I think it has more to do with frozen/non-frozen than transformer/non-transformer, but it's basically sane, and I'd rather not make the change here.